docs/src/test-api/class-test.md

```diff
@@ -1710,6 +1710,12 @@ Step body.
 
 Whether to box the step in the report. Defaults to `false`. When the step is boxed, errors thrown from the step internals point to the step call site. See below for more details.
 
+### option: Test.step.location
+* since: v1.50
+- `location` <[Object]>
+
+Specifies a custom location for a test step as { file, line, column }, enabling precise error reporting within user code.
+
```

Comment:
```suggestion
### option: Test.step.location
* since: v1.48
- `location` <[Location]>

Specifies a custom location for the step to be shown in test reports. By default, location of the [`method: Test.step`] call is shown.

```

Reply:
@dgozman Thanks for the quick response to my modified commit! I will immediately reflect this part of your answer :)